whetstone/src/main/java/com/deliveryhero/whetstone/logging/GlobalAndroidComponentListener.kt

```diff
@@ -0,0 +1,15 @@
+package com.deliveryhero.whetstone.logging
+
+import java.lang.IllegalStateException
+
+public object GlobalAndroidComponentListener {
+    public var componentInjectionListener: ComponentInjectionListener? = null
```

Comment:
then maybe instead of `vetoable` we can use `observable` delegate, imho it still looks more neat :) something like
```
public var componentInjectionListener: ComponentInjectionListener? by Delegates.observable(null) { _, old, new ->
    if (new != null && old != null) throw IllegalStateException("You are overriding an existing listener")
}
```

Reply:
also would be great to update the documentation, as Whetstone is already open source

Reply:
maybe creating wiki later for different part. I can add it there but I like the brief version for people. for now I added a little explanation on top of the class